### ISSUE TITLE
Request: Support for Faster Than Light; Nomad

### ISSUE THREAD

**Issue by MadBlue2112** [xdy/twodsix-foundryvtt#1567]
I don't know if this is too much of a departure from the 2D6 rules to be included, but it would be nice if Stellagama's newest game was able to be supported.

**Comment by marvin9257** [comment]
Yes, from what I understand the rolling mechanics are quite different.  Although, I don't have a copy of the rules yet. It will take a large rework of the roll code. I will try to investigate further after we move to v12.  Thanks

**Comment by marvin9257** [comment]
I asked Stellagama for permission.  They were very positive about this inclusion.

**Comment by marvin9257** [comment]
So I spent some time looking through the rules.  I'm not certain that Nomad can easily be incorporated into Twodsix.  It is possible, but there would be som many changes/exceptions I feel that it would almost be writing a system within a system.  Also, it would need new actor sheets as there are no characteristics.  While it would be great to include, I think it would be better implemented as its own system.

**Comment by marvin9257** [comment]
After a brief discussion with Stellagama, we agreed that it would be better for Nomad to have its own FVTT implementation rather than trying to integrate into Twodsix.  Twodsix would carry too much overhead for such a streamlined (and excellent) system.  Sorry.
